recipes/inner-html/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "inline-html" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "3e23c51b7d80211b4251e811b1c6f3d98a520185e91e1a55d5761619971e3e8f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - setup-py.patch
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
+    - cssutils
+    - click
+    - lxml
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - inline_html
+  commands:
+    - inline-html --help
+
+about:
+  home: https://github.com/zopyx/inline-html
+  license: GPL-2.0
+  license_family: GPL
+  # NB: Pending on upstream https://github.com/zopyx/inline-html/issues/2
+  # license_file: LICENSE.txt
```

Comment:
If this gets not merged, we need to include the license in the feedstock, GPL requires a copy of the license.

Reply:
No problem. Is this best done with a patch? Or is there dedicated functionality in conda for adding files?